ParisBNF/abb/BNFabb147.xml

```diff
@@ -187,24 +188,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                               </supportDesc>
 
                      <layoutDesc>
-                        <layout columns="2" writtenLines="16">
-                            <locus from="1" to="122"/>
-                        </layout>
+                       <layout columns="2" writtenLines="34 35">
+                       <locus from="ir" to="iv">i-ii</locus>
```

Comment:
i am not sure the roman numbering is accepted in foliation.
if i-ii are ff. 1-2 then all the other folio numbers must be checked

Reply:
We used to to this for DS and elsewhere, it seems to have worked.